### PR TITLE
Add a goimports verification

### LINKED ISSUE #78: add an imports verification
/kind feature

**Describe the solution you'd like**
We verify golint and gofmt but we don't test goimports

### PR DESCRIPTION
**Pull request by ericzhang08**
**What this PR does / why we need it**:add a  goimports verivication

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #78 

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by chuckha** [comment]
hmm i wonder what failed. I need to update that script to show me which script failed.

/test pull-cluster-api-provider-docker-verify

**Comment by chuckha** [comment]
Oh, I bet you go imports is not available in the test build. @ericzhang08 could you add a line to install the command, much like the speller does as well?

**Comment by ericzhang08** [comment]
@chuckha  ok, I am fixing it soon.

File: hack/verify-goimports.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+# cd to the root path
+cd_root_path
+
+# create a temporary directory
+TMP_DIR=$(mktemp -d)
+
+echo $TMP_DIR
```
Comment (ashish-amarnath) [inline hack/verify-goimports.sh]: looks like leftovers from debugging.

Comment (ericzhang08) [reply hack/verify-goimports.sh]: Thanks.I forgot it.

**Comment by ashish-amarnath** [comment]
Thank you @ericzhang08 for getting this done! :)

**Comment by chuckha** [comment]
/approve

thank you!

**Pull request merged**